### ISSUE TITLE
"alias-model-in-controller" should allow nested properties

### ISSUE THREAD

**Issue by buschtoens** [netguru/eslint-plugin-ember#79]
```js
export default Route.extend({
  store: service(),

  model() {
    const store = get(this, 'store');

    return hash({
      pets:   store.findAll('pet'),
      people: store.findAll('people')
    });
  }
});
```

```js
export default Controller.extend({
  pets:   reads('model.pets'),
  people: reads('model.people')
});
```

Loading multiple different models via `RSVP.hash` and aliasing them is a common pattern in Ember applications. Currently `alias-model-in-controller` does not detect this and prompts the user to alias `model`.
